Extension/src/LanguageServer/configurations.ts

```diff
@@ -386,6 +395,71 @@ export class CppProperties {
         }
     }
 
+    public nodeAddonIncludesFound(): number {
+        return this.nodeAddonIncludes.length;
+    }
+
+    private async readNodeAddonIncludeLocations(rootPath: string): Promise<void> {
+        let error: Error | undefined;
+        const package_json: any = await fs.promises.readFile(path.join(rootPath, "package.json"), "utf8")
+            .then(pdj => JSON.parse(pdj))
+            .catch(e => (error = e));
+
+        const nodeAddonMap: { [dependency: string]: string } = {
+            "nan": "node --no-warnings -e \"require('nan')\"",
+            "node-addon-api": "node --no-warnings -p \"require('node-addon-api').include\""
+        };
+
+        let added: number = 0;
```

Comment:
This isn't being used currently. Should it be removed?

Reply:
yes, nice eye - thanks. it was an experiment to avoid `handleConfigurationChange()` but it didn't work correctly.